guideline/cfh.md

```diff
@@ -131,26 +131,11 @@ off by the PAC before they can be put in place.
    Proposals will have to provide details about how these can be implemented at
    the proposed venues.
 
-   1. Food and drinks for lunches and breaks on all conference days (and
```

Comment:
I agree with about lunch.
But as you know there are many countries and different situation
In some venues, providing food cost much money, for example food for one person, one time can be at least 70 USD in some venue in korea. 
in that case, we should ask Participants to pay at least 150 USD ( at least 2 meals )
so I think it's case by case. 

I agree it's good and will to have, but it should not be necessary condition to remove hurdle.